src/Toml/Bi/Map.hs

```diff
@@ -506,3 +529,4 @@ _IntSet = iso IS.toList IS.fromList >>> _Array _Int
 
```

Comment:
I think `_Set`, `_HashSet` and `_IntSet` deserve the pragma as well :slightly_smiling_face: